bin/dev

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH.unshift(File.expand_path("../../lib", __FILE__))
+
+require "pry"
+require "doggy"
+Doggy::CLI.start(ARGV)
```

Comment:
it starts a console while this gives the same thing as running from `cli` + `pry`